tests/feehistory.rs

```diff
@@ -0,0 +1,125 @@
+use env_logger::Env;
+use eyre::Result;
+use helios::{config::networks::Network, prelude::*};
+use std::{env, path::PathBuf};
+
+#[tokio::test]
+async fn feehistory() -> Result<()> {
+    //Instantiate Client
+    env_logger::Builder::from_env(Env::default().default_filter_or("info")).init();
+
+    let api_key = env::var("YOUR_API_KEY").expect("Missing API key in environment variables, just run export YOUR_API_KEY='your-api-key-goes-here'");
```

Comment:
We already have an rpc url in our secrets called `MAINNET_RPC_URL`. If we switch the `untrusted_rpc_url` to that it should work.

Reply:
Switched